discord-bot/bot/extensions/work.py

```diff
@@ -105,16 +104,17 @@ async def _handle_task(ctx: lightbulb.SlashContext, task_type: TaskRequestType)
             else:
                 logger.critical(f"Unexpected task type received: {new_task.type}")
 
-        # Send a message in the log channel that the task is complete
-        # TODO: Maybe do something with the msg ID so users can rate the "answer"
-        assert ctx.guild_id is not None
+        # Send a message in all the log channels that the task is complete
         conn: Connection = ctx.bot.d.db
-        guild_settings = await GuildSettings.from_db(conn, ctx.guild_id)
+        async with conn.cursor() as cursor:
+            await cursor.execute("SELECT log_channel_id FROM guild_settings")
+            log_channel_ids = await cursor.fetchall()
+            print(log_channel_ids)
```

Comment:
logger?

Reply:
Good point.

There is a flake8 plugin that disallows print statements, it might be worth it to add that to the pre-commit.